File: Python/DewanPerwakilan.py
# deklarasi kelas
class DewanPerwakilan:

    # atribut private
    __id = ""
    __nama = ""
    __bidang = ""
    __partai = ""

   # konstruktor
    def __init__(self, id="", nama="", bidang="", partai=""):
        self.__id = id
        self.__nama = nama
        self.__bidang = bidang
        self.__partai = partai

    # get & set

    def get_id(self):
        return self.__id

    def set_id(self, id):
        self.__id = id

    def get_nama(self):
        return self.__nama

    def set_nama(self, nama):
        self.__nama = nama

    def get_bidang(self):
        return self.__bidang

    def set_bidang(self, bidang):
        self.__bidang = bidang

    def get_partai(self):
        return self.__partai

    def set_partai(self, partai):
        self.__partai = partai

File: PHP/DewanPerwakilan.php
<?php
// deklarasi kelas dpr
class DewanPerwakilan {
    // atribut private
    private $id = '';
    private $nama = '';
    private $bidang = '';
    private $partai = '';
    private $foto = '';

    // konstruktor
    public function __construct( $id = '', $nama = '' , $bidang = '', $partai = '', $foto='' ) {
        $this->id = $id;
        $this->nama = $nama;
        $this->bidang = $bidang;
        $this->partai = $partai;
        $this->foto = $foto;
    }
    
    // get & set id
    public function set_id($id){
        $this->id = $id;
    }

    public function get_id(){
        return $this->id;
    }

    // get & set nama
    
    public function set_nama($nama){
        $this->nama = $nama;
    }
    public function get_nama(){
        return $this->nama;
    }

    // get & set bidang

    public function set_bidang($bidang){
        $this->bidang = $bidang;
    }

    public function get_bidang(){
        return $this->bidang;
    }

    // get & set partai
    
    public function set_partai($partai){
        $this->partai = $partai;
    }
    public function get_partai(){
        return $this->partai;
    }

    // get & set partai

    public function set_foto($foto){
        $this->foto = $foto;
    }
    public function get_foto(){
        return $this->foto;
    }

}

?>